open-machine-learning-jupyter-book/ml-fundamentals/web-app.md

```diff
@@ -1 +1,346 @@
-# Web app
\ No newline at end of file
+# Web app
+# Build a Web App to use a ML Model
```

Comment:
I think let's use this as the title and you can update the file name to be consistent.

Also, make ML to be Machine Learning. And Model -> model